addons/knobs/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "license": "MIT",
   "main": "dist/index.js",
-  "jsnext:main": "src/index.js",
```

Comment:
`types` should be `dist/index.d.ts` not `src` because src is certainly not published 😉